src/main/steps/application-submitted/content.ts

```diff
@@ -1,142 +1,156 @@
-  line8: `If you want to ‘serve’ (send) the documents to your ${partner} yourself then phone ${config.get(
-    'servicePhoneNo'
-  )} to request it. Otherwise the court will do it.`,
-  line9: `If you want the court to serve (send) the application to be served by post instead of by email, then phone ${config.get(
-    'servicePhoneNo'
-  )}.`,
-  line10: `The address you have provided for your ${partner} is outside of England and Wales. That means you are responsible for ‘serving’ (sending) the court documents, which notify your ${partner} about ${
-    isDivorce ? 'the divorce' : 'ending the civil partnership'
-  }.`,
-  line11: `You will receive the documents that you need to send to your ${partner} by email and letter, after the application has been checked.`,
-  subHeading5: 'Dividing your money and property',
-  line12: {
-    part1: `It’s usually more straightforward and less expensive if you agree with your ${partner} on how to divide your money and property.`,
-    part2: 'Get help agreeing.',
-    link: `${config.get('govukUrls.mediation')}`,
-  },
-  line13: {
-    part1:
-      'If you do agree then you can make the agreement legally binding. This is known as asking the court to make a ',
-    part2: '‘consent order’',
-    link: `${config.get('govukUrls.consentOrder')}`,
-  },
-  line14: {
-    part1: 'If you disagree then you can ask the court to decide for you. This is known as applying for a ',
-    part2: '‘financial order’',
-    link: `${config.get('govukUrls.financialOrder')}`,
-  },
-  line15: {
-    part1: 'Read the guidance on ',
-    part2: 'money and property when you divorce or separate',
-    link: `${config.get('govukUrls.moneyAndProperty')}`,
-  },
-  subHeading6: 'If you need help',
-  line16: {
-    part1:
-      'Court staff can give you help with your application. They cannot give you legal advice. You should speak to a ',
-    linkText: 'solicitor or legal adviser',
-    link: `${config.get('govukUrls.legalAdvisor')}`,
-  },
-  webChat: 'Web chat',
-  webChatDetails: 'No agents are available, please try again later.',
-  sendUsAMessage: 'Send us a message',
-  telephone: 'Telephone',
-  telephoneNumber: `Telephone: ${config.get('servicePhoneNo')}`,
-  telephoneDetails: 'Monday to Friday 8am to 5pm',
-  telephoneCharges: {
-    part1: 'Find out about call charges',
-    link: `${config.get('govukUrls.callCharges')}`,
-  },
-  domesticAbuse: {
-    part1: 'If you are experiencing domestic abuse or feel unsafe, then ',
-    part2: 'support is available',
-    link: `${config.get('govukUrls.domesticAbuse')}`,
-  },
-  feedback: 'Help improve this service',
-  feedbackDetails: {
-    part1: 'This is a new service. ',
-    part2: 'Your feedback',
-    part3: ' helps to improve it for others.',
-    link: `${config.get('govukUrls.feedbackSurvey')}`,
-  },
-});
```

Comment:
how come this shows up as a complete rewrite? is it a formatting issue?

Reply:
it's because I'm now returning the content as an object so I only have to instantiate the const `currentState = currentStateFn(userCase);` once